src/functions/template-tags/day.php

```diff
@@ -1,23 +1,29 @@
 <?php
 if ( ! function_exists( 'tribe_is_day' ) ) {
 	/**
-	 * Single Day Test
+	 * Checks if the current view is the Day View.
 	 *
-	 * Returns true if the query is set for single day, false otherwise
+	 * This function determines whether the current view is the Day View by
+	 * comparing the current view slug with the Day View slug. It also allows
+	 * for the result to be filtered.
 	 *
-	 * @return bool
+	 * @return bool True if the current view is the Day View, false otherwise.
 	 */
 	function tribe_is_day(): bool {
 		$context  = tribe_context();
 		$is_day = tec_is_view( \Tribe\Events\Views\V2\Views\Day_View::get_view_slug() );
 
 		/**
-		 * Allows filtering of the tribe_is_day boolean value.
+		 * Filters the boolean value indicating if the current view is the Day View.
 		 *
-		 * @since 6.0.7 Added context to parameters.
```

Comment:
Why did we remove the `@since`?

Reply:
`@since` should have been moved below the description and above the `@param` list.

Reply:
I should move the `@since` but removed instead, my bad.